init.lua

```diff
@@ -109,15 +109,19 @@ There is also no flag for combat looting. It will only loot if no mobs are withi
 local mq = require 'mq'
 local success, Write = pcall(require, 'lib.Write')
 if not success then printf('\arERROR: Write.lua could not be loaded\n%s\ax', Write) return end
-
+local eqServer = string.gsub(mq.TLO.EverQuest.Server(),' ','_')
+local eqChar = mq.TLO.Me.Name()
+local version = 1.2
 -- Public default settings, also read in from Loot.ini [Settings] section
 local loot = {
     logger = Write,
-    Version = "1.0",
+    Version = '"'..tostring(version)..'"',
     LootFile = mq.configDir .. '/Loot.ini',
+    SettingsFile = mq.configDir.. '/LootNScoot_'..eqServer..'_'..eqChar..'.ini',
     AddNewSales = true,
     LootForage = true,
     DoLoot = true,
+    WasLooting = false,
```

Comment:
how does tracking this help across restarts? wouldn't you just not be starting/calling the script if you weren't supposed to be looting?

Reply:
In my kisscharm fork If people want to use lootnscoot. I have it call `/squelch /lua run lootnscoot once`. when its time to loot.  this works great until the bags are full.